sdk/include/ddk/ntstrsafe.h

```diff
@@ -2825,6 +2826,45 @@ RtlStringLengthWorkerW(
     return Status;
 }
 
+NTSTRSAFEAPI 
+RtlUnicodeStringPrintf(PUNICODE_STRING DestinationString,
+	NTSTRSAFE_PCWSTR pszFormat, ...)
```

Comment:
Please use correct coding style everywhere in your code:
- 4-space indentation;
- optionally for "if" blocks with 1-line case, surround around braces;
- The prototypes of the functions should be like:
```c
NTSTRSAFEAPI
RtlFunctionName(
  <SAL annotation> PUNICODE_STRING Parameter,
  <SAL annotation> TYPE OtherParameter)
```